frontend/src/components/stages/ranking_editor.scss

```diff
@@ -49,4 +49,10 @@ pr-icon-button {
 .indented {
   /* width of checkbox plus gap */
   margin-left: calc(48px + common.$spacing-medium);
+}
+
+.warning{
+  font-style: italic;
+  color: var(--md-sys-color-error);
+  width: 80%;
```

Comment:
Nit: use the common info-width variable (the same one used for stage descriptions, info/tos stages). I can push a fix for this.